### ISSUE TITLE
NVDA double speaks items in the Windows 11 task manager performance tab

### ISSUE THREAD

**Issue by enessaribas** [nvaccess/nvda#13252]
### Steps to reproduce:
1. insure that no other windows are open
2. open task manager, and navigate to performance tab
 3. tab around the items.
### Actual behavior:
NVDA speaks the first part of the item, for example perf performance tab, it is spoken very briefly and cut off, and following appears in log.
Input: kb(desktop):shift+control+escape
IO - speech.speech.speak (00:22:31.571) - MainThread (7452):
Speaking ['Task Manager', 'ProcessesPerformance App history Startup Users Details Services\ue70e49% 16% 0% 0%Name Status CPU Memory Disk Network\nFewer details End task', CancellableSpeech (still valid)]
IO - speech.speech.speak (00:22:31.703) - MainThread (7452):
Speaking ['Task Manager', CancellableSpeech (still valid)]
IO - speech.speech.speak (00:22:31.711) - MainThread (7452):
Speaking ['List of items', 'data grid', CancellableSpeech (still valid)]
IO - speech.speech.speak (00:22:31.743) - MainThread (7452):
Speaking ['List of items', 'list', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (00:22:32.190) - winInputHook (15256):
Input: kb(desktop):control+tab
IO - speech.speech.speak (00:22:33.277) - MainThread (7452):
Speaking ['Tab control', 'tab control', CancellableSpeech (still valid)]
IO - speech.speech.speak (00:22:33.293) - MainThread (7452):
Speaking ['Performance', 'tab', 'selected', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (00:22:34.112) - winInputHook (15256):
Input: kb(desktop):tab
IO - inputCore.InputManager.executeGesture (00:22:34.978) - winInputHook (15256):
Input: kb(desktop):tab
IO - speech.speech.speak (00:22:35.021) - MainThread (7452):
Speaking ['CPU', 'button', CancellableSpeech (still valid)]
IO - speech.speech.speak (00:22:35.044) - MainThread (7452):
Speaking ['CPU', 'button', 'CPU activity', CancellableSpeech (still valid)]
IO - speech.speech.speak (00:22:35.068) - MainThread (7452):
Speaking ['CPU', 'button', CancellableSpeech (still valid)]
DEBUG - speech.manager.SpeechManager._handleIndex (00:22:35.077) - MainThread (7452):
Unknown index 8, speech probably cancelled from main thread.

### Expected behavior:
NVDA speaks item only once.
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
alpha-24471,c6a6df82
#### Windows version:
Windows 11 21H2, also observed in Windows 10
#### Name and version of other software in use when reproducing the issue:
task manager
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by seanbudd** [comment]
Closing as a duplicate of #13250 and #11599
